src/test/scala/io/archivesunleashed/ArchiveRecordTest.scala

```diff
@@ -44,10 +45,68 @@ class ArchiveRecordTest extends FunSuite with BeforeAndAfter {
     assert(RecordLoader.loadArchives(warcPath, sc).count == 299L)
   }
 
+  test("Resource name produces expected result.") {
+    val textSampleArc = RecordLoader.loadArchives(arcPath, sc)
+     .map(x => FilenameUtils.getName(x.getResourcename))
+     .take(3)
+    val textSampleWarc = RecordLoader.loadArchives(warcPath, sc)
+     .map(x => FilenameUtils.getName(x.getResourcename)).take(3)
+    assert(textSampleArc.deep == Array("example.arc.gz",
+      "example.arc.gz", "example.arc.gz").deep)
+    assert(textSampleWarc.deep == Array("example.warc.gz",
+      "example.warc.gz", "example.warc.gz").deep)
+  }
+
+  test("Crawl Dates") {
+    val textSampleArc = RecordLoader.loadArchives(arcPath, sc)
+     .map(x => x.getCrawlDate).take(3)
+    val textSampleWarc = RecordLoader.loadArchives(warcPath, sc)
+     .map(x => x.getCrawlDate).take(3)
+    assert(textSampleArc.deep == Array("20080430", "20080430", "20080430").deep)
+    assert(textSampleWarc.deep == Array("20080430", "20080430", "20080430").deep)
+  }
+
+  test("Domains") {
```

Comment:
No other tickets.  Basically, last PR I had good test coverage, but failed to test particular cases and my code passed Travis with bugs.  I decided to include these additional tests in case that happened again (it was unlikely, but I wanted this PR to go more smoothly). Since ArchiveRecord is used widely across the tests, I did not expect the tests to improve coverage (as per #260).